### ISSUE TITLE
build with mingw-w64 (no msys)

### ISSUE THREAD

**Issue by sonoro1234** [pkulchenko/wxlua#22]
Hi Paul,

I used this branch in wxLua and pkulckendo/wxWidgets
WX_3_1_0-7d9d59

then compiled wxWidgets with
`mingw32-make -f makefile.gcc BUILD=release` and tested samples/minimal

and wxLua with
```
"cmake"  -G"MinGW Makefiles"  -DCMAKE_BUILD_TYPE="RelWithDebInfo" -DwxWidgets_ROOT_DIR="C:\supercolliderrepos/wxWidgetsZB" -DwxWidgets_LIB_DIR="C:\supercolliderrepos/wxWidgetsZB\lib\gcc_lib"  -DwxWidgets_CONFIGURATION=mswu -DBUILD_VERBOSELY=TRUE -DBUILD_SHARED_LIBS=FALSE -DwxLua_LUA_LIBRARY_USE_BUILTIN=FALSE  -DwxLua_LUA_LIBRARY=C:\luaGL\sources\Lua5.1\Lua51.dll -DwxLua_LUA_INCLUDE_DIR=C:\luaGL\gitsources\lua\src  -DwxWidgets_COMPONENTS="stc;gl;html;aui;adv;core;net;base" -DwxLuaBind_COMPONENTS="stc;gl;html;aui;adv;core;net;base" ../wxLua/wxLua
```

But there were building errors

```
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp: In function 'wxLuaBindNumber* wxLuaGetDe
fineList_wxcore(size_t&)':
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:530:31: error: 'wxADJUST_MINSIZE' was not
 declared in this scope
         { "wxADJUST_MINSIZE", wxADJUST_MINSIZE },
                               ^~~~~~~~~~~~~~~~
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:530:31: note: suggested alternative: 'wxF
IXED_MINSIZE'
         { "wxADJUST_MINSIZE", wxADJUST_MINSIZE },
                               ^~~~~~~~~~~~~~~~
                               wxFIXED_MINSIZE
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:654:30: error: 'wxBLIT_00220326' was not
declared in this scope
         { "wxBLIT_00220326", wxBLIT_00220326 },
                              ^~~~~~~~~~~~~~~
...
```
What have I done wrong?
Thanks

victor bombi

**Comment by sonoro1234** [comment]
May be it was related to compat28 so I edited setup.h in include/wx/msv with `#define WXWIN_COMPATIBILITY_2_8 1` and `#define WXWIN_COMPATIBILITY_3_0 1` before compiling wxWidgets

Now wxLua building error is

```
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp: In member function 'virtual bool wxLuaBi
nding_wxcore::RegisterBinding(const wxLuaState&)':
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:7758:46: error: 'wxEVT_COMMAND_DIRPICKER_
CHANGED' was not declared in this scope
     wxLua_wxEVT_COMMAND_DIRPICKER_CHANGED  = wxEVT_COMMAND_DIRPICKER_CHANGED;
                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:7758:46: note: suggested alternative: 'wx
EVT_COMMAND_FONTPICKER_CHANGED'
     wxLua_wxEVT_COMMAND_DIRPICKER_CHANGED  = wxEVT_COMMAND_DIRPICKER_CHANGED;
                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                              wxEVT_COMMAND_FONTPICKER_CHANGED
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:7759:46: error: 'wxEVT_COMMAND_FILEPICKER
_CHANGED' was not declared in this scope
     wxLua_wxEVT_COMMAND_FILEPICKER_CHANGED = wxEVT_COMMAND_FILEPICKER_CHANGED;
                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\supercolliderrepos\wxlua\wxLua\modules\wxbind\src\wxcore_bind.cpp:7759:46: note: suggested alternative: 'wx
EVT_COMMAND_FONTPICKER_CHANGED'
     wxLua_wxEVT_COMMAND_FILEPICKER_CHANGED = wxEVT_COMMAND_FILEPICKER_CHANGED;
                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                              wxEVT_COMMAND_FONTPICKER_CHANGED
```

**Comment by sonoro1234** [comment]
fixed in PR #23
